src/theory/quantifiers/ematching/inst_match_generator.cpp

```diff
@@ -87,175 +88,186 @@ int InstMatchGenerator::getActiveScore( QuantifiersEngine * qe ) {
     unsigned ngtt = qe->getTermDatabase()->getNumTypeGroundTerms( tn );
     Trace("trigger-active-sel-debug") << "Number of ground terms for " << tn << " is " << ngtt << std::endl;
     return ngtt;
-//  }else if( d_match_pattern_getKind()==EQUAL ){
-    
-  }else{
-    return -1;
   }
+  return -1;
 }
 
-void InstMatchGenerator::initialize( Node q, QuantifiersEngine* qe, std::vector< InstMatchGenerator * > & gens ){
-  if( !d_pattern.isNull() ){
-    Trace("inst-match-gen") << "Initialize, pattern term is " << d_pattern << std::endl;
-    if( d_match_pattern.getKind()==NOT ){
-      Assert(d_pattern.getKind() == NOT);
-      //we want to add the children of the NOT
-      d_match_pattern = d_match_pattern[0];
-    }
+void InstMatchGenerator::initialize(Node q,
+                                    QuantifiersEngine* qe,
+                                    std::vector<InstMatchGenerator*>& gens)
+{
+  if (d_pattern.isNull())
+  {
+    gens.insert(gens.end(), d_children.begin(), d_children.end());
+    return;
+  }
+  Trace("inst-match-gen") << "Initialize, pattern term is " << d_pattern
+                          << std::endl;
+  if (d_match_pattern.getKind() == NOT)
```

Comment:
Maybe use the `kind::` prefix throughout? You should also clean the `using namespace` commands above.